src/app/mail/mail-settings/mail-settings.component.ts

```diff
@@ -119,6 +128,18 @@ export class MailSettingsComponent implements OnInit, AfterViewInit, OnDestroy {
         this.changeUrlParams();
       }
     );
+
+    this.timeZoneFilteredOptions = this.timeZoneFilter.valueChanges
+      .pipe(
+        startWith(''),
+        map(name => name ? this._filterTimeZone(name) : this.timezones.slice())
+      );
+  }
+
+  private _filterTimeZone(name) {
+    const filterValue = name.toLowerCase();
```

Comment:
no need to declare `const filterValue` as it is used only on one place, use directly `name.toLowerCase(); `